packages/lit-ssr/src/lib/render-lit-html.ts

```diff
@@ -15,11 +15,11 @@
  */
 
 // Type-only imports
-import {TemplateResult, ChildPart} from 'lit-html';
+import {TemplateResult, ChildPart} from 'lit';
```

Comment:
You can use `import type` now to clarify this

Reply:
Done